src/parameterizations/vertical/MOM_internal_tide_input.F90

```diff
@@ -43,15 +43,15 @@ module MOM_int_tide_input
   type(diag_ctrl), pointer :: diag => NULL() !< A structure that is used to
                         !! regulate the timing of diagnostic output.
   real :: TKE_itide_max !< Maximum Internal tide conversion
-                        !! available to mix above the BBL [R Z3 T-3 ~> W m-2]
+                        !! available to mix above the BBL [H Z2 T-3 ~> W m-2]
```

Comment:
The units of TKE_itide_max and TKE_itidal _input vary depending on whether the model is Boussinesq, so they should be `[H Z2 T-3 ~>  m3 s-3 or W m-2]`